stock_picking_mass_action/models/stock_picking.py

```diff
@@ -10,9 +10,13 @@ class StockPicking(Model):
     _inherit = "stock.picking"
 
     @api.model
-    def check_assign_all(self):
+    def check_assign_all(self, other_domain=None):
```

Comment:
I would have said `domain` without 'other' as this will be the only one at the end